test/e2e/primaryoption.spec.ts

```diff
@@ -1,48 +1,12 @@
 /// <reference types="../../cypress/support" />
 
 describe("Primary option page", () => {
-  it("should show primary option page once", () => {
-    cy.login();
-    // WHEN I successfully login
-    cy.visit("/anmelden/erfolgreich");
-    // THEN I click continue button
-    cy.contains("a", "Verstanden & weiter").click();
-    // THEN I should see bundesident primary option page
-    cy.contains(
-      "h1",
-      "Möchten Sie sich in wenigen Minuten mit Ihrem Ausweis identifizieren?"
-    );
-    // THEN I click Identifikation mit Ausweis
-    cy.contains("a", "Identifikation mit Ausweis").click();
-    // WHEN I visit the success login page again
-    cy.visit("/anmelden/erfolgreich");
-    // THEN I click continue button
-    cy.contains("a", "Verstanden & weiter").click();
-    // THEN I should not see bundesident primary page
-    cy.contains("h1", "Mit welcher Option möchten Sie sich identifizieren?");
-    cy.url().should("include", "/identifikation");
-  });
-
```

Comment:
Why not keep the tests and just remove the second click? Is this already tested somewhere else?

Reply:
you are right! thank you!